lib/tether/toolbox.go

```diff
@@ -389,7 +391,7 @@ func mountDiskLabel(label string) (string, error) {
 
 	// label exists, bind mount it to a tmp directory
 	tmpDir, err := ioutil.TempDir("", fmt.Sprintf("toolbox-%s", label))
-	if err := Sys.Syscall.Mount(path, tmpDir, "bind", syscall.MS_BIND, ""); err != nil {
+	if err := Sys.Syscall.Mount(path, tmpDir, "ext4", syscall.MS_NOATIME, ""); err != nil {
```

Comment:
There is a const for this `ext4FileSystemType`

does remind me - we should be check that we're mounting with NOATIME in the offline path.